src/components/Dropdown.tsx

```diff
@@ -0,0 +1,85 @@
+import cn from 'classnames';
+import debounce from 'lodash.debounce';
```

Comment:
Also as recomendation - try to write own debounce function

![2023-09-25_17h01_01](https://github.com/mate-academy/react_autocomplete/assets/61904995/e8faacc9-3de5-4a39-ae14-5b25b6704db4)